lizard_connector/connector.py

```diff
@@ -169,13 +183,16 @@ def __init__(self, endpoint, base="https://demo.lizard.net", **kwargs):
                               obtained. When set to False only the first
                               page is obtained on get.
         """
-        super().__init__(**kwargs)
+        try:
+            super().__init__(**kwargs)
+        except:
+            super(Endpoint, self).__init__(**kwargs)
```

Comment:
`super(Endpoint, self).__init__(**kwargs)` werkt ook in python3 dus je hebt geen try except nodig. Test maar even uit.